test/e2e/scenarios/no_np.go

```diff
@@ -0,0 +1,18 @@
+// Test the scenario where there are no network policies
+// implemented in which case all pods should be able to
+// communicate with each other
+
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+	// . "github.com/onsi/gomega"
+)
+
+var _ = PDescribe("No network policy is implemented", func() {
+	PIt("Should allow communication between all pods", func() {
+		By("creating pod1 with label1 in cluster1 in namespace1")
+		By("creating pod2 with label2 in cluster2 in namespace2")
+		By("testing connectivity between pods")
+	})
+})
```

Comment:
Here you can add the negative test for network policy, where you create an empty policy, and try to connect between pods. This is helpful to identify if somebody is trying to run the tests without a valid (NP support) network plugin.

In such case we need to throw a huge warning explaining.